fcrepo-indexing-triplestore/src/main/java/org/fcrepo/camel/indexing/triplestore/FcrepoTripleStoreIndexingConfig.java

```diff
@@ -29,20 +29,20 @@
 import org.springframework.context.annotation.Configuration;
 
 /**
- * A configuration class for the triplestore Indexer service
+ * A configuration class for the triplestore Indexing service
  *
  * @author dbernstein
  */
 @Configuration
-@Conditional(FcrepoTripleStoreIndexerConfig.TriplestoreIndexerEnabled.class)
-public class FcrepoTripleStoreIndexerConfig extends BasePropsConfig {
+@Conditional(FcrepoTripleStoreIndexingConfig.TriplestoreIndexingEnabled.class)
+public class FcrepoTripleStoreIndexingConfig extends BasePropsConfig {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(FcrepoTripleStoreIndexerConfig.class);
-    static final String TRIPLESTORE_INDEXER_ENABLED = "triplestore.indexer.enabled";
+    private static final Logger LOGGER = LoggerFactory.getLogger(FcrepoTripleStoreIndexingConfig.class);
```

Comment:
This Logger isn't used so I opted to remove it in #175, can remove it here instead